RELEASE.md

```diff
@@ -103,31 +103,13 @@ The README file contains a section which lists all currently supported releases
 in a table. The version in this table needs to be updated to match the new
 release.
 
-## Create a GitHub release
+## Update the GitHub release notes
 
-It is better to have github create the final release tag, rather than pushing
-it through git. Pushing through git will auto-create an empty release and
-notify all the users before you have a chance to include the list of changes,
-or any other metadata.
+When a tag is pushed, a Github Action job takes care of creating a new Github
```

Comment:
Should we add the instructions to push the tag via git?

Reply:
I think there are instructions above in this file.

Reply:
I was wrong. I added a new step with instructions to create and push a tag.